internal/manager/task_scan_scene.go

```diff
@@ -82,7 +82,7 @@ func (t *ScanTask) associateCaptions(ctx context.Context) {
 				captions, er := sqb.GetCaptions(s.ID)
 				if er == nil {
 					if !scene.IsLangInCaptions(captionLang, captions) { // only update captions if language code is not present
-						newCaptions := scene.AddLangToCaptions(captionLang, captions)
+						newCaptions := append(captions, captionLang)
 						er = sqb.UpdateCaptions(s.ID, newCaptions)
```

Comment:
The newCaptions was needed because of AddLangToCaptions. Since a simple append is now enough we can use the below instead (not tested but should work and pass the gocritic check)
```golang
captions = append(captions, captionLang)
er = sqb.UpdateCaptions(s.ID, captions)
```

Reply:
Good catch. In my haste, this morning didn't bother to look down and see `newCaptions` was being used.